pacstrap/pacstrap.sh

```diff
@@ -9,19 +9,26 @@ fi
 
 pacman -Sy --noconfirm
 
+# kernel selection
+KERNEL="$(iroot kernel)"
+if ! [ KERNEL || KERNEL -eq "linux" || KERNEL -eq "linux-lts" ]; then
+    # fallbacks to lts kernel
+    KERNEL="linux-lts"
+fi
```

Comment:
this can be rewritten as
```sh
if iroot kernel; then
    KERNEL="$(iroot kernel)"
else
    # fallback to linux-lts
    KERNEL="linux-lts"
fi
```

Reply:
It looks 1000 times better, thank you

Reply:
The only problem is the "default" option, so KERNEL will be equals to "default" won't install any kernel

Reply:
Oh, I forgot about that

Reply:
How about
```sh
if iroot kernel; then
    KERNEL="$(iroot kernel)"
    if [ "$KERNEL" = "default" ]
    then
        KERNEL="linux-lts"
    fi
else
    # fallback to linux-lts
    KERNEL="linux-lts"
fi
```

Reply:
I thinks it's good